src/components/_inputs/TextField/TextField.tsx

```diff
@@ -32,27 +35,41 @@ const TextFieldComponent: React.ForwardRefRenderFunction<HTMLInputElement, TextF
     required,
     placeholder,
     defaultValue,
+    nodeStart,
+    nodeEnd,
     ...inputBaseProps
   },
   ref
 ) => {
+  const [nodeLeftRef, nodeLeftBounds] = useMeasure()
+  const [nodeRightRef, nodeRightBounds] = useMeasure()
```

Comment:
Why do we need to measure anything here? I would suspect we can just make the nodes part of the layout and the input will get sized automatically?

Reply:
I see now that this is because the nodes have `position: absolute;` but do they need to?

Reply:
I didn't find a good way to do it on the first try so I used chakra UI for inspiration and they had it using absolute.

For ref: https://chakra-ui.com/docs/form/input

Reply:
What's stopping us from using a flex row layout? The nodes could be placed at the ends and the middle text input would grow to fill the available space